### PR TITLE
DIGI 13 List all avalable taxpages

### PR DESCRIPTION
**Pull request by anebhargavkrishna**
DIGI 13 List all avalable taxpages

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+            try
+            {
+                taxPageLibrary = new TaxPageLibrary();
+                Task<HttpResponseMessage> response = taxPageLibrary.getAllTaxPages();
+                string data = response.Result.Content.ReadAsStringAsync().Result;
+                modelList = JsonConvert.DeserializeObject<List<TaxPageModel>>(data);
+            }
+            catch (Exception ex)
+            {
+                ViewBag.ErrorMessage = "API is not responding";
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs:29]: Change the Error message  to 'Something went wrong. Contact administrator..'

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxPageModel.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+
+namespace DigiOrderEnterpriseModel
+{
+    public class TaxPageModel
+    {
+        public Guid GUID { get; set; }
+        public string TITLE { get; set; }
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxPageModel.cs:10]: Missing Required tag

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs (hunk 1 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs]: Changed Error messaged

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxPageModel.cs (hunk 2 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxPageModel.cs:11]: No constraint In database,But added required tag to title

**Pull request merged**